rep-2000.rst

```diff
@@ -823,6 +823,119 @@ Package manager use for dependencies:
++==========================+=====================+===============+============================+================================+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 1        | All Platforms              | All Architectures              |
++--------------------------+---------------------+---------------+----------------------------+--------------------------------+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS   | Tier 1        | All Platforms              | All Architectures              |
++--------------------------+---------------------+---------------+----------------------------+--------------------------------+
+| rmw_connextdds           | RTI Connext         | Tier 1        | Ubuntu, Windows, and macOS | All Architectures except arm64 |
++--------------------------+---------------------+---------------+----------------------------+--------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS   | Tier 2        | All Platforms              | All Architectures              |
++--------------------------+---------------------+---------------+----------------------------+--------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.8
+
+
+Dependency Requirements:
+
++-------------+---------------------------------+-------------------------------------------------------------------+
+|             |        Required Support         |                        Recommended Support                        |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Package     |  Ubuntu Jammy  |  Windows 10**  |  RHEL 8  |    macOS**    | Debian Bullseye  |   OpenEmbedded**    |
++=============+================+================+==========+===============+==================+=====================+
+| CMake       |     3.21.3     |     3.22.0     |  3.18.2  |    3.14.4     |      3.18.4      | 3.16.1 / 3.12.2**** |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| EmPY        |                                                3.3.2                                                |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Gazebo      |    11.8.0*     |       N/A      |   N/A    |    11.0.0     |      11.0.0*     |         N/A         |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Ignition    |    Fortress*   |       N/A      |   N/A    |   Edifice*    |     Edifice*     |         N/A         |
```

Comment:
@chapulina Can you comment on what Gazebo and Ignition versions we might expect in Ubuntu 22.04?

Reply:
Yes! I've been waiting for this PR! It's going to be Gazebo 11.X and Ignition Fortress. We may be able to remove the `*` later, let's leave it there now to be safe.

Are the `x`s ok? In practice, the minor versions change over time.

```suggestion
| Gazebo      |    11.x.x*     |       N/A      |   N/A    |    11.x.x     |      11.x.x*     |         N/A         |
+-------------+----------------+----------------+----------+---------------+------------------+---------------------+
| Ignition    |    Fortress*   |       N/A      |   N/A    |   Fortress*    |     Fortress*     |         N/A         |
```

Reply:
Yeah, I think the Xs should be fine.